kobo/apps/project_ownership/utils.py

```diff
@@ -27,6 +28,14 @@ def create_invite(
     TransferStatus = apps.get_model('project_ownership', 'TransferStatus')
 
     with transaction.atomic():
+        # Set 'is_excluded_from_projects_list' to 'True' for assets owned by the
+        # sender or invitee joining the organization. These assets should be
+        # excluded from the organization owner's 'My Projects' list
+        real_owner = get_real_owner(sender)
+        for asset in assets:
+            asset.is_excluded_from_projects_list = real_owner != sender
+        Asset.objects.bulk_update(assets, ['is_excluded_from_projects_list'])
```

Comment:
This should be moved inside the "transfer_project()" method to respect atomicity (only per asset, no bulk_update). 
Moreover, "create_invite" is also used for regular project transfer ownership. 
This should happen only if the invite is a `OrgMembershipAutoInvite` instance.